[IMP] Inventory: Adding CMR to dispatch management system (#19081)
odoo/documentation: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst

In the *Operations* section, select the :guilabel:`Batch, Wave & Cluster Transfers` checkbox to
prepare batches of orders for delivery.

In the *Shipping* section, select the :guilabel:`Delivery Methods` and :guilabel:`Transport
Management` checkboxes. Doing so allows specific vehicles to be :ref:`set as carriers
<inventory/shipping_receiving/delivery-method-for-carrier>`.

> In the *Warehouse* section, select the :guilabel:`Storage Locations` checkbox to assign specific

Reviewer: optional, but support have told me that particularly for these setting checkboxes, customers prefer to see screenshots! how about adding one here?

<img width="1176" height="198" alt="Image" src="https://github.com/user-attachments/assets/0b5102f8-f4e8-46ed-8928-56bb67f9ce4e" />

.. image:: dispatch/google-map.png
   :alt: Show Google Map route.

Prepare consignment note (CMR)
==============================

The CMR convention (Convention on the Contract for the International Carriage of Goods by Road) is a

> convention concerning transportation of cargo by road. The majority of European states have ratified

Reviewer 2: ```suggestion
The CMR convention (Convention on the Contract for the International Carriage of Goods by Road) governs the transportation of cargo by road. The majority of European states have ratified
```

the convention.

A standard waybill or transport document was developed to aid in this treaty. This is called a

> consignment note (or a CMR waybill). Senders are required to produce these forms. A waybill consists

Reviewer 2: ```suggestion
consignment note (also known as a CMR waybill). Senders are required to produce these forms. A waybill consists
```

of four parts:

#. Sender's copy (in red)
#. Receiver's copy (in blue)
#. Carrier's copy (in green)
#. Second carrier's copy (in black)

> Odoo completes these forms with information it has; other information must be added manually.

Reviewer 2: ```suggestion
Odoo completes these forms using the information it has; any additional information must be added manually.
```

Single-delivery CMR
-------------------

Print a CMR for a delivery order by navigating to :menuselection:`Inventory app --> Operations -->
Deliveries`. Open the delivery order that requires a consignment note.

Click the :icon:`fa-cog` :guilabel:`(Actions)` icon, then select :icon:`fa-print`
:menuselection:`Print --> Consignment Note (CMR)`. A consignment note is generated.

.. image:: dispatch/cmr-note.png
   :alt: A CMR note for a shipment of produce, going to Belgium.

Batch delivery CMR
------------------

Print CMRs for a batch order by navigating to :menuselection:`Inventory app --> Operations -->

> Batch Transfers`. Open the batch transfer that requires consignment notes.

Reviewer 2: ```suggestion
Print CMRs for a batch order by navigating to :menuselection:`Inventory app --> Operations --> Batch Transfers`. Open the batch transfer that requires consignment notes.
```
Double check line length

.. image:: dispatch/google-map.png
   :alt: Show Google Map route.

Prepare consignment note (CMR)
==============================

> The CMR convention (Convention on the Contract for the International Carriage of Goods by Road)

Reviewer: what a mouthful

governs the transportation of cargo by road. The majority of European states have ratified the
convention.

A standard waybill or transport document was developed to aid in this treaty. This is called a

> consignment note (also known as a CMR waybill). Senders are required to produce these forms. A

Reviewer: starting from the first time you defined the abbr CMR, let's use the `:abbr:` tag. abbreviations give keyword relevance to search and LLMs!

since the fully spelled out version is really long and breaks up text, feel free to use a `replace::` at the beginning of the article! 

for convenience, this is the syntax:

```
.. |CMR| replace:: :abbr:`CMR (whatever long form it is)`
```
